docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
     image: postgres
     container_name: pg-api-starter
     ports:
-      - '5400:5432'
```

Comment:
Let's change this back to 5400 for the repo. I think we updated this for your PC only.